index.html

```diff
@@ -787,101 +748,37 @@ <h4>Spatial Simulcast and Temporal Scalability</h4>
-        attribute is defined in this specification.
+        This is an example of {{MediaCapabilities/encodingInfo(configuration)}}
+        returned by a browser implementing [[WEBRTC]] and [[Media-Capabilities]].
       </p>
        <pre class="example highlight">
-[
-    {
-      "clockRate": 90000,
-      "mimeType": "video/VP8",
-      "scalabilityModes": [
-        "L1T1",
-        "L1T2",
-        "L1T3"
-      ]
-    },
-    {
-      "clockRate": 90000,
-      "mimeType": "video/rtx"
-    },
-    {
-      "clockRate": 90000,
-      "mimeType": "video/VP9",
-      "scalabilityModes": [
-        "L1T1",
-        "L1T2",
-        "L1T3"
-      ],
-      "sdpFmtpLine": "profile-id=0"
-    },
-    {
-      "clockRate": 90000,
-      "mimeType": "video/VP9",
-      "sdpFmtpLine": "profile-id=2"
-    },
-    {
-      "clockRate": 90000,
-      "mimeType": "video/H264",
-      "sdpFmtpLine": "level-asymmetry-allowed=1;packetization-mode=1;profile-level-id=42001f"
-    },
-    {
-      "clockRate": 90000,
-      "mimeType": "video/H264",
-      "sdpFmtpLine": "level-asymmetry-allowed=1;packetization-mode=0;profile-level-id=42001f"
-    },
-    {
-      "clockRate": 90000,
-      "mimeType": "video/H264",
-      "sdpFmtpLine": "level-asymmetry-allowed=1;packetization-mode=1;profile-level-id=42e01f"
-    },
-    {
-      "clockRate": 90000,
-      "mimeType": "video/H264",
-      "sdpFmtpLine": "level-asymmetry-allowed=1;packetization-mode=0;profile-level-id=42e01f"
-    },
-    {
-      "clockRate": 90000,
-      "mimeType": "video/AV1",
-      "scalabilityModes": [
-        "L1T1",
-        "L1T2",
-        "L1T3",
-        "L2T1",
-        "L2T1h",
-        "L2T1_KEY",
-        "L2T2",
-        "L2T2_KEY",
-        "L2T2_KEY_SHIFT",
-        "L3T1",
-        "L3T3",
-        "L3T3_KEY",
-        "S2T1"
-      ]
-    },
-    {
-      "clockRate": 90000,
-      "mimeType": "video/H264",
-      "sdpFmtpLine": "level-asymmetry-allowed=1;packetization-mode=1;profile-level-id=4d001f"
-    },
-    {
-      "clockRate": 90000,
-      "mimeType": "video/H264",
-      "sdpFmtpLine": "level-asymmetry-allowed=1;packetization-mode=1;profile-level-id=64001f"
-    },
-    {
-      "clockRate": 90000,
-      "mimeType": "video/red"
-    },
-    {
-      "clockRate": 90000,
-      "mimeType": "video/ulpfec"
-    }
-]
+const contentType = 'video/VP9';
+
+const configuration = {
+  type: 'webrtc',
+  video: {
+    contentType: contentType,
+    width: 640,
+    height: 480,
+    bitrate: 10000,
+    framerate: 29.97,
+    scalabilityMode: "L3T3_KEY"
+  }
+};
+
+navigator.mediaCapabilities.encodingInfo(configuration)
+  .then((result) => {
+    console.log(contentType + ' is:'
+      + (result.supported ? '' : ' NOT') + ' supported,'
+      + (result.smooth ? '' : ' NOT') + ' smooth and'
+      + (result.powerEfficient ? '' : ' NOT') + ' power efficient');
+  })
+  .catch((err) => {
+    console.error(err, ' caused encodingInfo to reject');
+  });
```

Comment:
The WebRTC spec uses modern JS like `async`/`await` in examples. We may want to stay consistent with that.
Regarding output, if it's unsupported, it probably doesn't matter whether it's choppy or slow. How about:  
```js
const contentType = 'video/VP9';

const configuration = {
  type: 'webrtc',
  video: {
    contentType,
    width: 640,
    height: 480,
    bitrate: 10000,
    framerate: 29.97,
    scalabilityMode: "L3T3_KEY"
  }
};

try {
  const info = await navigator.mediaCapabilities.encodingInfo(configuration);

  if (!info.supported) {
    console.log(`${contentType} is unsupported.`);
    return;
  }
  console.log(`${contentType} is ${info.smooth || 'NOT '}smooth and` +
              `${info.powerEfficient || 'NOT '}power efficient`);
} catch (err) {
  console.error(err, ' caused encodingInfo to fail');
}
```

Reply:
Fixed.